### PR TITLE
Add more description on doc.go of the `common` package

### PR DESCRIPTION
**Pull request by andybclee**
## Proposed changes

- This PR adds more description of `common` package on its doc.go

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: common/doc.go [hunk 1]
```diff
@@ -18,5 +18,24 @@
...
+And also it provides cache and helper functions.
+
+Source Files
+
+Each source files provides functions and types as listed below
+  - big.go   	: Defines common big integers often used such as Big1
+  - bytes.go	: Provides conversion functions between a byte slice with a string in Hex format
+  - cache.go	: Defines Cache interface and constants such as CacheType
+  - debug.go	: Provides a function to let an user know where to report a bug. It is being used by datasync/downloader
+  - format.go	: Provides function to print a time.Duration in more readable format
```
Comment (ehnuje) [inline common/doc.go]:
```suggestion
  - format.go	: Provides a function to print a time.Duration in more readable format
```

File: common/doc.go [hunk 2]
```diff
@@ -18,5 +18,24 @@
...
+  - path.go  	: Provides functions to check a file's existence and to get an absulute path
```
Comment (ehnuje) [inline common/doc.go]:
```suggestion
  - path.go  	: Provides functions to check a file's existence and to get an absolute path
```

File: common/doc.go [hunk 3]
```diff
@@ -18,5 +18,24 @@
...
+
+The `common` package defines commonly used data types and constants.
+
+And also it provides cache and helper functions.
+
+Source Files
+
+Each source files provides functions and types as listed below
+  - big.go   	: Defines common big integers often used such as Big1
+  - bytes.go	: Provides conversion functions between a byte slice with a string in Hex format
```
Comment (ehnuje) [inline common/doc.go]:
```suggestion
  - bytes.go	: Provides conversion functions between a byte slice and a string in Hex format
```

File: common/doc.go [hunk 4]
```diff
@@ -18,5 +18,24 @@
 // This file is derived from common/bytes.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package common contains various helper functions.
+/*
+Package common contains various helper functions.
+
+The `common` package defines commonly used data types and constants.
+
+And also it provides cache and helper functions.
```
Comment (ehnuje) [inline common/doc.go]: Please consider merging above three sentences into one like below =)

Package common contains various helper functions, commonly used data types and constants.

File: common/doc.go [hunk 5]
```diff
@@ -18,5 +18,24 @@
...
+
+Source Files
+
+Each source files provides functions and types as listed below
```
Comment (laggu) [inline common/doc.go]:
```suggestion
Each source file provides functions and types as listed below
```

File: common/doc.go (hunk 4 above)
Comment (andybclee) [reply common/doc.go]: @ehnuje Good suggestion. Thanks!

**Pull request merged**